test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -1362,6 +1363,33 @@ static void test_sa_info(void)
 
 	ipsec_sa_destroy(sa_out);
 	ipsec_sa_destroy(sa_in);
+
+	/*
+	 * Additional check for SA lookup parameters. Let's use transport
+	 * mode SA and ODP_IPSEC_DSTADD_SPI lookup mode.
+	 */
+	ipsec_sa_param_fill(&param_in,
+			    true, false, 123, NULL,
+			    ODP_CIPHER_ALG_AES_CBC, &key_a5_128,
+			    ODP_AUTH_ALG_SHA1_HMAC, &key_5a_160,
+			    NULL, NULL);
+	param_in.inbound.lookup_mode = ODP_IPSEC_LOOKUP_DSTADDR_SPI;
+	param_in.inbound.lookup_param.ip_version = ODP_IPSEC_IPV4;
+	param_in.inbound.lookup_param.dst_addr = &dst;
+	sa_in = odp_ipsec_sa_create(&param_in);
+	CU_ASSERT_FATAL(sa_in != ODP_IPSEC_SA_INVALID);
+
+	memset(&info_in, 0, sizeof(info_in));
+	CU_ASSERT_FATAL(odp_ipsec_sa_info(sa_out, &info_in) == 0);
```

Comment:
I guess this should be sa_in.

Reply:
Indeed. Fixed in v2.

I was surprised that the test passed with that bug but it turned out sa_in handle got the same value (SA table idx) as sa_out.